source/reference/image-types.rst

```diff
@@ -51,26 +51,15 @@ Table 2 lists the currently available images that are platform specific.
    * - aws.img
      - Image suitable for use with Amazon\* AWS\*.
 
-   * - azure.vhd
-     - Virtual Hard Disk for use on Microsoft\* Azure\* cloud platform.
-
-   * - azure-docker.vhd
-     - Virtual Hard Disk for use on Microsoft Azure cloud platform with
-       Docker\* pre-installed.
-
-   * - azure-machine-learning.vhd
-     - Virtual Hard Disk for use on Microsoft Azure cloud platform with the
-       `machine-learning-basic` bundle installed.
-
    * - cloudguest.img
      - Image with generic cloud guest virtual machine (VM) requirements
        installed.
 
    * - gce.tar
      - Image with the Google Compute Engine (GCE) specific kernel.
 
-   * - hyperv.vhdx
-     - Virtual Hard Disk for use with Microsoft Hyper-V\* hypervisor. Includes
+   * - azure-hyperv.vhd
+     - Image for Microsoft* Azure and Hyper V generation 1 VMs. Includes
```

Comment:
I had Noelia add a dash between Hyper and V on clearlinux.org/downloads.  Let's match :)
```
Image for Microsoft* Azure and Hyper-V generation 1 VMs
```

Reply:
Will do. NP.